### PR TITLE
update readme and quick start guide

### PR DESCRIPTION
**Pull request by lee19830115**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,46 @@
...
-_NOTE_: This integration is exclusively tested for `core-image-base`
+- WINSYSTEM ITX-P-C444 (imx8mq-itx-p-c444) - [ITX-P-C444](https://www.winsystems.com/product/itx-p-c444/)
 
-```bash
-bitbake core-image-base
-```
+## Quick Start Guide
 
-**core-image-base**: "A console-only image that fully supports the target device hardware."
+See the Quick Start Guide for instructions of building core image and for a quick demo of **DM-Crypt with CAAM's black key**.
```
Comment (danie007) [inline README.md]: Can we link the `Quick Start Guide` in here?

File: README.md [hunk 2]
```diff
@@ -1,87 +1,46 @@
...
-#
 # See LICENSE file for license details.
 -->
 
 # meta-bgn-essa
 
 [BG Network's](https://bgnet.works/) [Embedded Security Software Architecture](https://bgnet.works/embedded-security-software-architecture/) (ESSA), a collection of scripts, recipes, configurations, and documentation for Linux, enhances cybersecurity for IoT devices, including secure boot, encryption, authentication, and secure software updates. The ESSA enables engineers to extend a hardware root of trust to secure U-Boot, the Linux kernel, and applications in the root file system.
 
-This repository is based on [WinSystems/c444-manifest](https://github.com/bgnetworks/c444-manifest/tree/master) (_5.4.47 release_) and enables [Mender](https://mender.io/) OTA and NXP's HAB features on WINSYSTEM's [ITX-P-C444](https://www.winsystems.com/product/itx-p-c444/) hardware.
+To provide strong cybersecurity without compromising performance or functionality, this architecture leverage:
```
Comment (hellbent) [inline README.md]:
```suggestion
To provide strong cybersecurity without compromising performance or functionality, this architecture leverage:s
```

**Review by hellbent — CHANGES_REQUESTED** [review]
Looks good, thanks! Minor nit below but otherwise happy to merge if you agree.

**Review by danie007 — CHANGES_REQUESTED** [review]
The changes look good and organised 👍 

I've gone through your [Quick start guide](https://github.com/bgnetworks/meta-bgn-essa/blob/zeus-w-caam-review/docs/Quick%20Start%20Guide.pdf) `PDF` and it cover the details. But **GitHub** does not natively supports PDF and a `markdown` file would be perfect in here. If possible, could you convert the doc to `.md`?

---

CC @tkmozhi

File: README.md (hunk 1 above)
Comment (hellbent) [reply README.md]: Great point @danie007! Maintaining that in markdown will make things easier to maintain moving forward. You can use something like https://word2md.com/ on the source document to make the conversion a little easier.

Comment (lee19830115) [reply README.md]: I added some images in the quick start guide. Haven't figured out a way to convert it to a markdown file. If you guys know how to handle it, let me know. I agree with PDF isn't an elegant solution. It is a temporary solution for now until I find a better file format. Will add a link for the quick start guide.

**Comment by lee19830115** [comment]
> The changes look good and organised +1
> 
> I've gone through your [Quick start guide](https://github.com/bgnetworks/meta-bgn-essa/blob/zeus-w-caam-review/docs/Quick%20Start%20Guide.pdf) `PDF` and it cover the details. But **GitHub** does not natively supports PDF and a `markdown` file would be perfect in here. If possible, could you convert the doc to `.md`?
> 
> CC @tkmozhi

I will take a look if it is possible to create quick start guide in markdown files

**Pull request merged**